File: codingtest/java/template_stdin.java
// 표준 입출력형 템플릿 (백준 / 일부 코테) — Java.
// 같은 이름 입력파일(prob.txt/.in) 또는 input.txt를 만들면 run.sh가 stdin으로 넣어준다.
//
// 입력은 BufferedReader + StringTokenizer, 출력은 StringBuilder 한 번에 flush 하는 것이
// Java 코테의 기본 (Scanner/System.out.println 남발은 시간초과의 주범).

import java.io.*;
import java.util.*;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        int n = Integer.parseInt(br.readLine().trim());
        StringTokenizer st = new StringTokenizer(br.readLine());

        long sum = 0;
        for (int i = 0; i < n; i++) {
            sum += Long.parseLong(st.nextToken());
        }

        sb.append(sum).append('\n');
        System.out.print(sb); // 마지막에 한 번에 출력
    }
}

File: codingtest/cpp/template_stdin.cpp
// 표준 입출력형 템플릿 (백준 / 일부 코테).
// input.txt를 만들어 두면 자동으로 그 파일을 입력으로 읽는다 (run.sh가 처리).
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int n;
    cin >> n;
    vector<int> a(n);
    for (int& x : a) cin >> x;

    long long sum = 0;
    for (int x : a) sum += x;
    cout << sum << "\n";

    return 0;
}
